frontend/src/Link/index.js

```diff
@@ -13,10 +13,11 @@ function Link(props) {
   if (!node) {
     return <a href={props.href}>{props.children}</a>;
   }
-  let hash = props.href.split('?t=')[1] || undefined;
+  let url = new URL(props.href, window.location.origin);
+  let params = Object.fromEntries(new URLSearchParams(url.search));
```

Comment:
`params` is such a loaded word in React context, maybe you can find a different one.

Reply:
Can you add an comment, on what is happening here and why? Without knowing about URL and friends it's not immdiately clear to me and probably others.